pyQuARC/schemas/rule_mapping.json

```diff
@@ -3715,6 +3715,34 @@
         "severity": "error",
         "check_id": "one_item_presence_check"
     },
+    "license_url_description_check": {
+        "rule_name": "License URL Description Check",
+        "fields_to_apply": {
+            "echo-c": [
+                {
+                    "fields": [
+                        "Collection/UseConstraints/LicenseURL/Description"
+                    ]
+                }
+            ],
+            "dif10": [
+                {
+                    "fields": [
+                        "DIF/Use_Constraints/License_URL/Description"
+                    ]
+                }
+            ],
+            "umm-c": [
+                {
+                    "fields": [
+                        "UseConstraints/LicenseURL/Description"
+                    ]
+                }
+            ]
+        },
+        "severity": "warning",
+        "check_id": "one_item_presence_check"
+    },
```

Comment:
This check should only go through if the license information check above is satisfied through the LicenseURL/URL field instead of the LicenseText field. Perhaps we should add the LicenseURL/URL field for each schema here as well.

Reply:
After testing, I think we actually want the License URL description recommendation to appear if LicenseText is not provided. (i.e. it would not depend on whether the License URL/URL was provided or not). Would that be difficult to implement?